src/services/roster/index.js

```diff
@@ -0,0 +1,166 @@
+'use strict';
+const hooks = require("./hooks");
+const globalHooks = require('../../hooks');
+const oauth2 = require('../oauth2/hooks');
+
+module.exports = function() {
+	const app = this;
+
+	app.use('/roster', {
+		find(params) {
+			return Promise.resolve("Roster interface available");
+		}
+	})
+
+	const metadataHandler = {
+		find(params) {
+			return app.service("pseudonym").find({
+				query: {
+					pseudonym: params.pseudonym
+				}
+			}).then(pseudonym => {
+				if (!pseudonym.data[0]) {
```

Comment:
You can use `pseudonym.total` or append `paginate: false` after the query and use `pseudonym.length`